storage/types.go

```diff
@@ -318,10 +318,10 @@ func (n *NodeID) Split(prefixBytes, suffixBits int) ([]byte, Suffix) {
 
 	bits := n.PrefixLenBits - prefixBytes*8
 	if bits > suffixBits {
-		panic(fmt.Sprintf("storage Split: %x(n.PrefixLenBits: %v - prefixBytes: %v *8) > %v", n.Path, n.PrefixLenBits, prefixBytes, suffixBits))
+		panic(fmt.Sprintf("storage Split: %x(n.PrefixLenBits: %v - prefixBytes: %v*8) > %v", n.Path, n.PrefixLenBits, prefixBytes, suffixBits))
 	}
 	if bits == 0 {
```

Comment:
Might this be better as `bits <= 0`?  Currently your added test for this case (`TestSplitPanics/splitoutofbounds`) panics with `slice bounds out of range` from line 329 below.

Reply:
Done.